### PR TITLE
Cirnia mission tree update for big map

### PR DESCRIPTION
**Pull request by That-Guy-Github1**
update for Cirnia's mission tree, did some minor alterations to the map and province history to make the tree make more sense.

### CODE REVIEW

File: missions/CIN_Missions.txt [hunk 1]
```diff
@@ -1243,21 +1435,25 @@ cin_slot4_branch = {
...
 			OR = {
-				area = Northern_fairy_land
-				area = Fairy_land
+				region = fairy_region
 			}
-			NOT = { development = 20 }
+			NOT = { OR = {
+			num_of_times_improved_by_owner = 5
+			development = 20				
+			} }
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: OR block should be in separate line

File: missions/CIN_Missions.txt [hunk 2]
```diff
@@ -1243,21 +1435,25 @@ cin_slot4_branch = {
...
 			country_or_non_sovereign_subject_holds = ROOT
 		}
 		trigger = {
 			num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
 				OR = {
-					area = Northern_fairy_land
-					area = Fairy_land
+					region = fairy_region
 				}
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Deprecated OR block

File: missions/CIN_Missions.txt [hunk 3]
```diff
@@ -1243,21 +1435,25 @@ cin_slot4_branch = {
...
 				culture_group = th_fairy_culture_g
-				development = 20
-				value = 6
+				OR = {
+			num_of_times_improved_by_owner = 4 #added line to hopefully help players that remove dev
+			development = 20				
+			}
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Indentation issues

File: missions/CIN_Missions.txt [hunk 4]
```diff
@@ -1273,15 +1469,21 @@ cin_slot4_branch = {
 		provinces_to_highlight = {
 			province_id = 4977
 			OR = {
-				NOT = { grown_by_development = 5 }
+				NOT = {	OR = { 
+				grown_by_development = 5
+				num_of_times_improved_by_owner = 6
+				} }
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Formatting

File: missions/CIN_Missions.txt [hunk 5]
```diff
@@ -1273,15 +1469,21 @@ cin_slot4_branch = {
...
 		}
 		trigger = {
 			custom_trigger_tooltip = {
 				tooltip = CIN_council_of_nine_mission_mystia
 				4977 = {
-					grown_by_development = 5
+					OR = { 
+				grown_by_development = 5
+				num_of_times_improved_by_owner = 6 # same reasoning as last time
+				}
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Formatting (indentation)

File: missions/CIN_Missions.txt [hunk 6]
```diff
@@ -1313,13 +1529,27 @@ cin_slot4_branch = {
...
+				area = himiko_area
+				area = crane_wing_refuge_area
+				area = shions_rest_area
+				area = hundred_oni_roads_area
+				area = windspire_area
+				area = serenity_falls_area
+				area = hakurei_western_coast
+				area = kirin_fields_area
+				area = yamatai_area
+				area = court_of_the_miko_area
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Formatting (indentation)

File: missions/CIN_Missions.txt [hunk 7]
```diff
@@ -1379,13 +1682,22 @@ cin_slot4_branch = {
...
+						area = onbashira_graveyard
+						area = southern_moriya_2
+						area = southern_moriya_3
+						area = phantom_meadow
+						area = wind_gods_lake
+						area = moriya_ropeway
+						area = moriya_coast_state_2
 					}
-					country_or_non_sovereign_subject_holds = ROOT
+					country_or_non_sovereign_subject_holds = ROOTt
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Should be ROOT not ROOTt

File: missions/CIN_Missions.txt [hunk 8]
```diff
@@ -1584,9 +1896,7 @@ cin_slot5_branch = {
 		position = 6
 		provinces_to_highlight = {
 			OR = {
-				area = Forest_of_magic
-				area = Forest_edge_state
-				area = Forest_coast_state
+				region = eastern_forest_region
 			}
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Deprecated OR block

File: events/flavorCIN.txt [hunk 9]
```diff
@@ -1157,15 +1157,43 @@ country_event = {
 			add_base_manpower = 1
 			change_culture = ROOT
 		}
-		Scarlet_island = {
-			limit = {
+		scarlet_gate_area = {
+				limit = {
```
Comment (Hecatia-Lapislazuli) [inline events/flavorCIN.txt]: Formatting (Indentation)

File: events/flavorCIN.txt [hunk 10]
```diff
@@ -1157,15 +1157,43 @@ country_event = {
...
+			izayoi_area = {
+				limit = {
+				country_or_non_sovereign_subject_holds = ROOT
+			}
+			add_province_modifier = {
+				name = cin_resettling_fairies
+				duration = 9125
+			}
+			}
+
```
Comment (Hecatia-Lapislazuli) [inline events/flavorCIN.txt]: Formatting (indentation)

File: events/flavorCIN.txt [hunk 11]
```diff
@@ -1189,15 +1217,42 @@ country_event = {
 		4968 = {
 			add_core = ROOT
 		}
-		Scarlet_island = {
-			limit = {
+			scarlet_gate_area = {
+				limit = {
```
Comment (Hecatia-Lapislazuli) [inline events/flavorCIN.txt]: Formatting (indentation)

File: events/flavorCIN.txt [hunk 12]
```diff
@@ -1189,15 +1217,42 @@ country_event = {
...
+			}
+			izayoi_area = {
+				limit = {
+				country_or_non_sovereign_subject_holds = ROOT
+			}
+			add_province_modifier = {
+				name = cin_resettling_fairies
+				duration = 9125
+			}
+			}
```
Comment (Hecatia-Lapislazuli) [inline events/flavorCIN.txt]: Formatting (indentation)

File: missions/CIN_Missions.txt [hunk 13]
```diff
@@ -609,19 +719,27 @@ cin_slot2_branch = {
...
 			NOT = { country_or_non_sovereign_subject_holds = ROOT }
 		}
 		trigger = {
 			num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
-				OR = {
-					area = Scarlet_mountain
-					area = Scarlett_domain
+				AND = {
+					region = misty_lake_region
+					NOT = { is_permanent_claim = ROOT }
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: This condition feels weird, and will behave weirdly. Can work for what provinces to highlight (if that's what you want), but here we shouldn't have it due to funky interactions with previously disqualified provinces becoming qualified again once we core them.

File: missions/CIN_Missions.txt [hunk 14]
```diff
@@ -1161,25 +1360,18 @@ cin_slot4_branch = {
 		required_missions = {  }
 		position = 2
 		provinces_to_highlight = {
-			OR = {
-				area = Fairy_land
-				province_id = 5083
-				province_id = 5084
-			}
+				region = fairy_region
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Formatting (indentation)

File: missions/CIN_Missions.txt [hunk 15]
```diff
@@ -1243,21 +1435,25 @@ cin_slot4_branch = {
 		position = 4
 		provinces_to_highlight = {
 			OR = {
-				area = Northern_fairy_land
-				area = Fairy_land
+				region = fairy_region
 			}
```
Comment (Hecatia-Lapislazuli) [inline missions/CIN_Missions.txt]: Deprecated OR block

**Review by Hecatia-Lapislazuli — CHANGES_REQUESTED** [review]

File: missions/CIN_Missions.txt (hunk 13 above)
Comment (Hecatia-Lapislazuli) [reply missions/CIN_Missions.txt]: @That-Guy-Github1 I did address most of the formatting stuff you missed, but still would want to bring this one up regarding having `NOT = { is_permanent_claim = ROOT }` before approving

It should be removed from the trigger, but whether to remove from province highlighting is up to you. Am just not sure if this should be replaced with other triggers in its place. (such as areas which got permanent claims from prior events)

**Comment by Hecatia-Lapislazuli** [comment]
Am still just waiting on this
https://github.com/Touhou-Universalis/tohouniv_updated/pull/167#discussion_r1621504556

> I did address most of the formatting stuff you missed, but still would want to bring this one up regarding having `NOT = { is_permanent_claim = ROOT }` before approving
> 
> It should be removed from the trigger, but whether to remove from province highlighting is up to you. Am just not sure if this should be replaced with other triggers in its place. (such as areas which got permanent claims from prior events)

**Comment by That-Guy-Github1** [comment]
> Am still just waiting on this [#167 (comment)](https://github.com/Touhou-Universalis/tohouniv_updated/pull/167#discussion_r1621504556)
> 
> > I did address most of the formatting stuff you missed, but still would want to bring this one up regarding having `NOT = { is_permanent_claim = ROOT }` before approving
> > It should be removed from the trigger, but whether to remove from province highlighting is up to you. Am just not sure if this should be replaced with other triggers in its place. (such as areas which got permanent claims from prior events)

oops sorry didn't see that fixed now

**Review by Hecatia-Lapislazuli — APPROVED** [review]

**Pull request merged**